src/components/ui/AddressCombobox.tsx

```diff
@@ -121,28 +123,33 @@ export const AddressCombobox = ({
         onKeyDown={(e) => handleEnter(e)}
         autoComplete={'off'}
       />
-      <ComboboxOptions
-        anchor="bottom"
-        className="address-listbox utrecht-listbox utrecht-listbox--html-div fixed z-[9999] pointer-events-auto"
-      >
-        <div className={'utrecht-listbox__list'}>
-          {addressOptions.length > 0 ? (
-            addressOptions.map((address) => (
+      {!loading && (
+        <ComboboxOptions
+          anchor="bottom"
+          className="address-listbox utrecht-listbox utrecht-listbox--html-div fixed z-[9999] pointer-events-auto"
+        >
+          <div className={'utrecht-listbox__list'}>
+            {addressOptions.length > 0 ? (
+              addressOptions.map((address) => (
+                <ComboboxOption
+                  key={address.id}
+                  value={address}
+                  className="utrecht-listbox__option data-[focus]:bg-blue-100 !px-3 !py-1"
```

Comment:
All these tailwind classes should be brand specific and set by using NL Design System.

I've added a proposal with https://github.com/delta10/signalen-frontend-wcag/pull/332

Reply:
volgens mij zijn we nu wel wat styling kwijt en is de visuele hover met keyboard select ook weg. 

daarnaast is de functionaliteit in de kaart dialoog helemaal kapot heb ik het idee